pkg/p2p/libp2p/connections_test.go

```diff
@@ -346,11 +340,11 @@ func TestBlocklisting(t *testing.T) {
 	expectPeers(t, s2)
 	expectPeersEventually(t, s1)
 
-	// s2 connects to s1, thus the notifier on s1 should be called on Connect
 	_, err = s1.Connect(context.Background(), addr2)
 	if err == nil {
 		t.Fatal("expected error during connection, got nil")
 	}
+	t.Logf("s1 connect threw error: %v", err)
```

Comment:
r: In the tests, this message looks like a false positive:

```sh
=== RUN   TestBlocklisting
    connections_test.go:347: s1 connect threw error: connect full close stream reset
--- PASS: TestBlocklisting (2.41s)
PASS
```

It is really needed here? Maybe we should rather assert this error or rewrite the message in a clearer way.

Reply:
woops. leftovers :) removing. thanks!